x-pack/plugins/security_solution/public/cases/components/case_view/index.tsx

```diff
@@ -40,6 +41,16 @@ import {
   normalizeActionConnector,
   getNoneConnector,
 } from '../configure_cases/utils';
+import { useQueryAlerts } from '../../../detections/containers/detection_engine/alerts/use_query';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { SignalHit, Signal } from '../../../../server/lib/detection_engine/signals/types';
```

Comment:
we can not do that, you can not import server side code in public. this will break the build. if you really need it move the types in common folder